cartography/data/jobs/cleanup/gcp_compute_vpn_gateways_cleanup.json

```diff
@@ -0,0 +1,17 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:GCPVpnGateway) WHERE n.project_id = {GCP_PROJECT_ID} AND n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
Would it be a lot of work to draw a path from the GCPVpnGateway to the GCPProject here instead of querying for n.projectid?

Reply:
I wanted to do the PROJECT ID standard part of the query in the cleanup the same way we do with the AWS account. This is to avoid the issue we have where we cleanup all the nodes of projects that haven't being synched yet on this run. I could probably make the way back to the GCPProject but I find this way more straightforward and easy to understand what's going on. Thoughts?

Reply:
@achantavy thoughts on this?

Reply:
I think we should follow the pattern in AWS modules where we check for 

```
(:GCPVpnGateway)<-[r:RESOURCE]-(:GCPProject)
```

I claim full responsibility for messing that up with the first GCP modules - sorry!

But yeah, all assets should be linked to an account/project/whatever-it-is-called-by-that-data-provider.